src/components/UserMapMarker.css

```diff
@@ -1,7 +1,5 @@
-.Event-dot {
-  width: '15px';
-  height: '15px';
-  background-color: #fc6888;
-  borderRadius: '10px';
-  box-shadow: 8px 8px rgba(255, 255, 255, 0.4)
+.User-dot {
```

Comment:
Also here prefix the classname with `UserMapMarker` so `.UserMapMarker-dot`. Otherwise when we create a User component or something like that the CSS would collide and we dig out for the error